libraries/AP_Logger/AP_Logger_File.cpp

```diff
@@ -85,6 +85,7 @@ void AP_Logger_File::Init()
     }
     if (bufsize >= _writebuf_chunk && bufsize != desired_bufsize) {
         DEV_PRINTF("AP_Logger: reduced buffer %u/%u\n", (unsigned)bufsize, (unsigned)desired_bufsize);
+        _front._params.file_bufsize.set(bufsize);
```

Comment:
why change the parameter? we try to avoid doing this as it means if the user extracts params then reloads later they end up with the lower parameter

Reply:
ok